setup.py

```diff
@@ -45,15 +44,18 @@ def install(package):
         "sphinx",
         "sphinx_rtd_theme",
         "versioneer",
-    ],
+    ]
 }
 
-# Ensure pytorch is already installed (see e.g.
-# https://github.com/pyg-team/pytorch_geometric/issues/861#issuecomment-566424944)
-try:
-    import torch  # pyright: reportMissingImports=false
-except ImportError:
-    install("torch==1.9.0")
+# https://pypi.org/classifiers/
+CLASSIFIER = {
+    "Development Status :: 3 - Alpha",
+    "Programming Language :: Python :: 3.7"
```

Comment:
Right now we are testing for python versions 3.6-3.9; but I have considered going to 3.7-3.10. For now, I think we should add 3.6 to this list, and then we can always updated if we decide to support a different set of versions.